spec/models/topic_spec.rb

```diff
@@ -599,18 +599,14 @@ def build_topic_with_title(title)
       expect(topic.reload.moderator_posts_count).to eq(1)
     end
 
-    context "when moderator post fails to be created" do
-      before do
-        user.toggle!(:blocked)
-      end
-
-      it "should not increment moderator_posts_count" do
-        expect(topic.moderator_posts_count).to eq(0)
-
-        topic.add_moderator_post(user, "winter is never coming")
+    it "creates a moderator post ignoring validations" do
```

Comment:
@tgxworld I missed your point here, sorry. Even if this actually modifies the expectation of this test, it also includes some cosmetic changes. I removed the comestic changes in #4863!